toolkit/tools/pkg/imagecustomizerlib/customizeboot.go

```diff
@@ -301,54 +365,54 @@ func updateSELinuxCommandLine(selinuxMode imagecustomizerapi.SELinuxMode, imageC
 	return nil
 }
 
-func updateSELinuxCommandLineHelper(grub2Config string, selinuxMode imagecustomizerapi.SELinuxMode) (string, error) {
-	// Remove all the existing SELinux args.
-	grub2Config, securityIndex, err := tryRemoveKernelCommandLineArgumentValue(grub2Config, "security")
-	if err != nil {
-		return "", err
-	}
-
-	grub2Config, selinuxIndex, err := tryRemoveKernelCommandLineArgumentValue(grub2Config, "selinux")
+// Finds all the kernel command-line args that match the provided names, then insert replacement arg(s).
+func updateKernelCommandLineArguments(grub2Config string, argsToRemove []string, newArg string) (string, error) {
```

Comment:
It is more consistent if the interface accepts either arrays or strings when expecting arguments/parameters. I'd vote for arrays as they are structured and help us avoid the need to re-parse the contents.

Reply:
`newArgs` is now a string array.